### PR TITLE
localnet: use 27000 port for prometheus

### PR DESCRIPTION
**Pull request by cmwaters**
## Description

This relates to #5805 and changes the port from :9090 to :27000 as :9090 is the default port that the actual prometheus server runs on. 

In order to do this, I have shifted the rpc and p2p ports of the other three nodes down

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ services:
     container_name: node1
     image: "tendermint/localnode"
     ports:
-      - "26659-26660:26656-26657"
```
Comment (melekes) [inline docker-compose.yml:24]: I'd rather change the prometheus port than break existing setup

Comment (cmwaters) [reply docker-compose.yml]: Okay I have changed it to 26665 or do you have a specific preference for which port it should be?

Comment (melekes) [reply docker-compose.yml]: not really, but better to use one from different range, say `27000`

**Review by melekes — APPROVED** [review]
👍

**Pull request merged**